docs/en/mnemonics/mymonero.md

```diff
@@ -0,0 +1,9 @@
+---
+title: MyMonero Mnemonic Scheme
+---
+
+The MyMonero scheme can be thought as the 13-word version of the [Legacy Mnemonic Scheme](./legacy.md). It was initially used by MyMonero web and mobile wallet, but other projects also adopted this scheme. The scheme is designed to be more user-friendly and easier to remember than the legacy scheme. The MyMonero scheme is also compatible with the legacy scheme, meaning that you can restore a wallet generated by the legacy scheme using the MyMonero scheme and vice versa.
```

Comment:
```suggestion
The MyMonero scheme can be thought of as the 13-word version of the [Legacy Mnemonic Scheme](./legacy.md). It was initially used by MyMonero web and mobile wallet, but other projects also adopted this scheme. The scheme is designed to be more user-friendly and easier to remember than the legacy scheme. The MyMonero scheme is also compatible with the legacy scheme, meaning that you can restore a wallet generated by the legacy scheme using the MyMonero scheme and vice versa.
```

Reply:
citation(s) required for "MyMonero scheme being compatible with legacy / restoring a wallet generated by the legacy scheme using the MyMonero scheme and vice versa".